config.tutorial.yaml

```diff
@@ -337,18 +337,36 @@ costs:
 
 
 monte_carlo:
```

Comment:
For reproducibility, it may be good to add an optional option on the seed, like "seed: 42".

Reply:
Absolutely